site/content/docs/alerting/ssl-expiration.md

```diff
@@ -28,6 +28,12 @@ Simply create or pick an existing alert channel that your check subscribes to an
 ## API checks
 The domain for the certificate is parsed from the `URL` in the HTTP request settings so it does not require any setup.
 
+
+>**Error: unable to verify the first certificate.**
+If prompted with this error, the usual cause is the certificate chain of the given website being incomplete. This will not happen with a browser check, because the browser will complete the certificate chain on its own. When running an API check, though, no browser is involved - therefore the error takes place.
+You can use an online SSL checker (e.g.: [https://www.sslshopper.com/ssl-checker.html](https://www.sslshopper.com/ssl-checker.html)) to help you diagnose issues with your certificate.
```

Comment:
Let's make that link proper... something like [SSLHopper](https://www.sslshopper.com/ssl-checker.html).